app/models/demo.rb

```diff
@@ -0,0 +1,2 @@
+class Demo < ApplicationRecord
```

Comment:
Where is `Demo` used?  I don't see a table for it

Reply:
That was a mistake.

Reply:
:+1: